src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -188,6 +188,9 @@ def publish_results(self, endpoint_buffer):
 
         self._handle_redis_job_metadata()
         self.redis_db.publish(self.result_channel, results)
+        endpoint_buffer.producer.send(
+            'd3a-results', value=json.dumps(result_report).encode('ascii'),
```

Comment:
2 comments here:
1. it would be better to have a separate constant for 'd3a-results' , for instance KAFKA_SIM_RESULTS_TOPIC
2. is the ascii encoding required here ? If not, why not using utf-8 as the encoding type?